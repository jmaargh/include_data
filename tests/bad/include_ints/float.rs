use include_data::include_ints;

fn main() {}

// Can't use this macro for floats
static FLOATS: &[f32] = include_ints!(f32, "../../test_data/binary_64");
