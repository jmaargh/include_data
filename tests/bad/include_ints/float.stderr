error[E0308]: mismatched types
 --> tests/bad/include_ints/float.rs:6:25
  |
6 | static FLOATS: &[f32] = include_ints!(f32, "../../test_data/binary_64");
  |                         ^^^^^^^^^^^^^^---^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |                         |             |
  |                         |             expected due to this
  |                         expected `f32`, found integer
  |                         help: use a float literal: `0.0`
  |
  = note: this error originates in the macro `include_ints` (in Nightly builds, run with -Z macro-backtrace for more info)
